src/renderer/store/editor.js

```diff
@@ -589,7 +601,19 @@ const actions = {
   },
 
   // This events are only used during window creation.
-  LISTEN_FOR_BOOTSTRAP_WINDOW ({ commit, state, dispatch }) {
+  LISTEN_FOR_BOOTSTRAP_WINDOW ({ commit, state, dispatch, rootState }) {
+    // Delay load runtime commands and initialize commands.
```

Comment:
Why do we need to delay load runtime commands?

Reply:
So all modules are loaded and all values are set in store but this also speed up startup time because we don't need this setup right at after startup.